Specify game version support on the docs

haloss1 opened the pull request:
I added a table to specify which Halo releases Chimera currently supports, i also changed chimeras description to give hint that it supports multiple releases of the game.

haloss1 committed:
Changed Chimera description to emphasize multiple game releases support

haloss1 committed:
Added Game version support table

haloss1 committed:
Merge branch 'master' into specify-version-support

BinToss commented on README.md:
```diff
@@ -1,26 +1,36 @@
+| Halo Trial                        | halod     | ✅ (01.00.00.0578) |
+| Halo Beta                         | halo      |         ❌         |
+| Halo Mac                          | halomac   |         ❌         |
+| Halo Trial (Mac)                  | halomacd  |         ❌         |
```
I thought this was typically shortened to 'halomd'.

BinToss replied:
Ah, my bad. [HaloMD refers to a mod for Halo Mac Demo](https://www.halomd.net/).

haloss1 replied:
Yes, i'm using the code names as how the games were registered in the gamespy master server
halor = Halo Retail (Combat evolved)
halom = Halo Multiplayer Expansion (Custom edition)
halod = Halo Demo (Trial)
halo = Halo Beta

Vaporeon commented:
The Mac version of the game also uses the same "codenames" as the Windows versions. I think this part can just be removed. Windows XP will also not work with provided builds because the version of winpthreads provided by the compiler does not support it.

haloss1 committed:
Merge branch 'SnowyMouse:master' into specify-version-support

haloss1 committed:
Clarify Windows XP requirements

haloss1 committed:
Fix Windows XP issue link

haloss1 committed:
Add ARM information

haloss1 commented:
Sorry, i've been busy with work, @Vaporeon i used the codenames that were registered to GameSpy as reference (Except MCC which i just used the shortname that pretty much everyone uses) according to this config file from aluigi's `gslist`: http://aluigi.altervista.org/papers/gslist.cfg, i've included the mac versions since they are pretty much another release of the game different from the windows releases.
As for Windows XP support i added your info and linked #80 as reference for people who would like to run chimera on XP.
And i also added ARM to the supported list of CPUs (I tested it on my Nintendo Switch a few months ago using box86 and Wine)

pR0Ps commented:
Can you make it so you're not modifying lines unrelated to your changes? It makes reviewing the PR harder and the resulting git history messier.

haloss1 commented:
Sure, sorry for [this](https://github.com/SnowyMouse/chimera/pull/77/commits/a7dc627fe3b982f8f7d06eb326ffdbc2e6c7ea2e), i ran the formatter on save, the rich diff should filter out all of the formatting changes

Keith94 commented:
Huh so does Chimera actually support the original Combat Evolved release?

Weird because on chimera.opencarnage.net it states something else, or maybe the text is outdated.

> Q: Is the retail version of Halo supported?
> A: No. Only Halo Custom Edition is supported.

Aerocatia commented:
AFAIK when Open Carnage changed hosts access to that page has been lost, so it can not be updated.
